cfme/tests/distributed/test_appliance_replication.py

```diff
@@ -45,14 +45,15 @@ def get_ssh_client(hostname):
 # and we should be using the with context manager, they are being used incorrectly below
 def stop_db_process(address):
     with get_ssh_client(address) as ssh:
-        assert ssh.run_command('service {}-postgresql stop'.format(db.postgres_version()))[0] == 0,\
+        assert ssh.run_command('service {}-postgresql stop'.format(
```

Comment:
OPTIONAL: service is sysV command. since our containers are RHEL based,  I'd recommend you to use systemd commands instead - systemctl, etc.